scripts/ansible/roles/public-ami/templates/avalanchego.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Avalanche go client
+After=syslog.target network.target
+
+[Service]
+User=avalanche
+Type=simple
+Environment=HOME=/home/avalanche
+ExecStart=/usr/local/bin/avalanchego --config-file /etc/avalanchego/conf.json
```

Comment:
i guess these 2 also would have to be updated/renamed. although I don't when and if they are ever used tbh

Reply:
those lines came as they were in camino-node, I suggest we adress such stuff systematically later, there are more of these